### PR TITLE
complete a few todos

### PR DESCRIPTION
**Pull request by devincd**
### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by fewdan** [comment]
You can exec "make check" first.

**Comment by devincd** [comment]
> You can exec "make check" first.

Hi, @fewdan I just re-goimport the file and it was successful.

File: cmd/chaos-builder/main.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var (
-	log = zap.Logger(true)
+	log = zap.New(zap.UseDevMode(true))
```
Comment (WangXiangUSTC) [inline cmd/chaos-builder/main.go:30]: why need to do this change?

Comment (devincd) [reply cmd/chaos-builder/main.go:30]:
```golang
// Deprecated: use New() and the functional opts pattern instead:
//
// New(UseDevMode(development))
func Logger(development bool) logr.Logger {
	return LoggerTo(os.Stderr, development)
}
```

File: controllers/podchaos/podfailure/types.go [hunk 2]
```diff
@@ -238,13 +258,16 @@ func (r *endpoint) recoverPod(ctx context.Context, pod *v1.Pod, podchaos *v1alph
...
+		key := utils.GenAnnotationKeyForImage(podchaos, name)
 
 		if pod.Annotations == nil {
 			pod.Annotations = make(map[string]string)
 		}
-
-		// FIXME: Check annotations and return error.
+		// Check annotations and return error
+		if _, ok := pod.Annotations[key]; !ok {
+			r.Log.Error(errNotOperatedChaos, "the pod not operated by podChaos", "namespace", pod.Namespace, "name", pod.Name)
```
Comment (WangXiangUSTC) [inline controllers/podchaos/podfailure/types.go:268]: how about using `continue` and just print a warn log?

for example, container 1 is not operated, and container2 is operated, these codes will return an error(this error will be ignored later), but the pod will not delete, the container2 can not be recovered.

File: cmd/chaos-builder/main.go (hunk 1 above)
Comment (WangXiangUSTC) [reply cmd/chaos-builder/main.go:30]: is it suggested to set `development` = true?

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

**Review by devincd — COMMENTED** [review]
Signed-off-by: devincd <505259926@qq.com>

**Comment by WangXiangUSTC** [comment]
😅 why close this pr

**Comment by devincd** [comment]
Hi, @WangXiangUSTC  It's getting a little messy, so let me PR again

**Comment by WangXiangUSTC** [comment]
ok, thanks

**Pull request closed without merging**